safe-global/safe-react
# Tx Simulation

## What it solves
Resolves #3952 

## How this PR fixes it

Enables simulation for transactions via Tenderly.

## How to test it

1. Open the safe app
2. Create Transaction, Sign or Execute an existing Transaction, BatchExecution
3. In the Review screen press Simulate
4. Observe a message showing up
6. Press the Tenderly link

## Screenshots

![Screenshot 2022-06-24 at 16 26 41](https://user-images.githubusercontent.com/5880855/175556460-c89a242a-ba12-4511-912b-573e52926b9d.png)
![Screenshot 2022-06-24 at 16 26 45](https://user-images.githubusercontent.com/5880855/175556466-25abf795-4d75-43f1-b986-266d049959f3.png)
![Screenshot 2022-06-24 at 16 27 03](https://user-images.githubusercontent.com/5880855/175556472-5825b908-7f17-458f-a230-c2366614c700.png)
